analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -113,18 +111,11 @@ class ClearlyDefinedPackageCurationProvider(server: Server = Server.PRODUCTION)
             return emptyList()
         }
 
-        val declaredLicenseParsed = curation.licensed?.declared?.let { declaredLicense ->
-            // Only take curations of good quality (i.e. those not using deprecated identifiers) and in particular none
-            // that contain "OTHER" as a license, also see https://github.com/clearlydefined/curated-data/issues/7836.
-            runCatching { declaredLicense.toSpdx(SpdxExpression.Strictness.ALLOW_CURRENT) }.getOrNull()?.toString()
-        }
-
         val sourceLocation = curation.described?.sourceLocation.toArtifactOrVcs()
 
         val pkgCuration = PackageCuration(
             id = pkgId,
             data = PackageCurationData(
-                declaredLicenses = declaredLicenseParsed?.let { sortedSetOf(it) },
```

Comment:
We cannot drop this without a substitute, but assign it to `concludedLicense` instead.